curvefs/src/client/fuse_s3_client.cpp

```diff
@@ -67,12 +67,14 @@ CURVEFS_ERROR FuseS3Client::Init(const FuseClientOption &option) {
 
     auto s3Client = std::make_shared<S3ClientImpl>();
     s3Client->Init(opt.s3Opt.s3AdaptrOpt);
+    const uint64_t writeCacheMaxByte =
```

Comment:
check the config value >= 8M or not, if not, exit fuse client.

Reply:
add a `#define MINBUFSIZE 8M` above

Reply:
Fixed.